### PR TITLE
Add CLI command for better LMDB compaction

### PR DESCRIPTION
**Pull request by SergiySW**
Docs change for https://github.com/nanocurrency/nano-node/pull/2435

### CODE REVIEW

File: docs/commands/command-line-interface.md [hunk 1]
```diff
@@ -122,6 +122,9 @@ Clear record history for long term online weight trending
 ### --peer_clear
 Clear cached peers
 
+### --rebuild_database
+Rebuild LMDB database with `--vacuum` for best compaction. Requieres approximately `data.ldb size * 2` free space on disk
```
Comment (wezrule) [inline docs/commands/command-line-interface.md]: Small typo: `Requieres`

Comment (SergiySW) [reply docs/commands/command-line-interface.md]: Thanks

File: docs/commands/command-line-interface.md [hunk 2]
```diff
@@ -131,6 +134,7 @@ Clear unchecked blocks
 
 ### --vacuum
 Compact database. If data_path is missing, the database in data directory is compacted. Optional `--unchecked_clear`, `--clear_send_ids`, `--online_weight_clear`, `--peer_clear`
+Optional `--rebuild_database` in version 21.0+
```
Comment (zhyatt) [inline docs/commands/command-line-interface.md]: Do we need to call out the *2 disk space here as well?

Comment (wezrule) [reply docs/commands/command-line-interface.md]: Should this be mentioned on `--snapshot` section as well? Also `--confirmation_height_clear` should probably be mentioned in vacuum too.

**Comment by zhyatt** [comment]
@SergiySW Are there additional details you need on the feedback above?

File: docs/commands/command-line-interface.md (hunk 2 above)
Comment (SergiySW) [reply docs/commands/command-line-interface.md:137]: Why mention this in `--snapshot`?

**Pull request merged**